applications/openshift/controller/controller_rotate_kubelet_server_certs/rule.yml

```diff
@@ -5,11 +5,12 @@ title: 'Ensure that the RotateKubeletServerCertificate argument is set'
 description: |-
     To enforce kublet server certificate rotation on the Controller Manager,
     edit the Controller Manager pod specification file
-    <tt>/etc/kubernetes/manifests/kube-controller-manager.yaml</tt> on the
-    master node(s) and set the <tt>--feature-gates</tt> parameter to include
+    <tt>/etc/origin/master/master-config.yaml</tt> on the
+    master node(s) and set the <tt>controllerArguments</tt> parameter to include
     <tt>RotateKubeletServerCertificate=true</tt>. For example:
-
-    <pre>--feature-gates=...,RotateKubletServerCertificate=true,...</pre>
+    <pre>kubernetesMasterConfig:
+      controllerArguments:
+        - RotateKubeletServerCertificate=true</pre>
```

Comment:
This isn't right. This is a `feature-gate`. See https://docs.openshift.com/container-platform/3.10/admin_guide/disabling_features.html

Reply:
thanks!